src/olympia/reviewers/forms.py

```diff
@@ -130,8 +128,9 @@ def clean_addon_type_ids(self):
 
     def filter_qs(self, qs):
         data = self.cleaned_data
-        if data['admin_review']:
-            qs = qs.filter(admin_review=data['admin_review'])
+        if data['needs_admin_code_review']:
+            qs = qs.filter(
+                needs_admin_code_review=data['needs_admin_code_review'])
```

Comment:
scope creep, but should we allow filtering by `needs_admin_content_review` too?

Reply:
Maybe in a follow-up, yes. It's unclear what are the exact needs regarding this.